cmd/zc_traverser_local.go

```diff
@@ -105,7 +146,7 @@ func WalkWithSymlinks(fullPath string, walkFunc filepath.WalkFunc) (err error) {
 			computedRelativePath = strings.TrimPrefix(computedRelativePath, common.AZCOPY_PATH_SEPARATOR_STRING)
 
 			if fileInfo.Mode()&os.ModeSymlink != 0 {
-				result, err := filepath.EvalSymlinks(filePath)
+				result, err := UnfurlSymlinks(filePath)
```

Comment:
Does this also need to be called in the non-recursive codepath, below?

Reply:
Yes it did, I've added the call.